transitions/center_wipe.h

```diff
@@ -0,0 +1,88 @@
+#ifndef TRANSITIONS_CENTER_WIPE_H
+#define TRANSITIONS_CENTER_WIPE_H
+
+#include "wave.h"
+#include "../functions/mult.h"
+#include "../functions/sum.h"
+
+// Usage: TrCenterWipeX<POSITION_FUNCTION, MILLIS_FUNCTION>
+// or: TrCenterWipeX<POSITION, MILLIS>
+// POSITION_FUNCTION & MILLIS_FUNCTION: FUNCTION
+// POSITION: Int
+// MILLIS: a number
+// return value: TRANSITION
+// In the beginning entire blade is color A, then color B 
+// starts at the POSTION and extends up and down the blade
+// in the specified number of milliseconds.
+template<class MILLIS, class POSITION = Int<16384>>
+class TrCenterWipeX : public TransitionBaseX<MILLIS> {
+public:
+  void run(BladeBase* blade) {
+    pos_.run(blade);
+    TransitionBaseX<MILLIS>::run(blade);
+    center_ = (pos_.getInteger(0) * blade->num_leds()) >> 7;
+    fade_top_ = this->update(256 * blade->num_leds() - center_);
```

Comment:
center_, fade_top_ and fade_bottom_ is only used in this function, so they should be local variables.

Reply:
fixed

Reply:
did you forget to upload or something? It doesn't look fixed.

Reply:
yeah, looks like I pushed a previous, working on the change you suggested but still getting compile error, sending in PM on TRA as it's odd.